Group Contribution

LalithaBanda opened the pull request:
Developed the fallowing Test Cases in Catalog Module , 
- Check_SplitOrder_AsPerItemType
- Verify_BackButton_CartCheckConfirmRequest
- Verify_ConfirmRequest_AddMoreItems
- Verify_ConfirmRequest_AttachExternalFile
- Verify_ConfirmRequest_AttachInternalFile
- Verify_ConfirmRequestPage_AddComments
- Verify_ConfirmRequestPage_GrandTotal
- Verify_CheckOutDetails_AddMoreItems
- Verify_CheckOutDetails_Confirm
- VerReq_CartCheckOutWithSameOrDifferntCC
- VerifyReq_CopyItemsForRFQCancelledStatus
- VerifyReq_SearchRejectedOrders

LalithaBanda committed:
Developed new Test "Verify_ConfirmRequestPage_AddComments"

LalithaBanda committed:
Added new Methods

LalithaBanda committed:
Added verification

PraveenNamburi committed:
1. Developed a new test in catalog module -
VerifyReq_DeleteExistingItemsFromCart.
2. Added the following methods in CartInfoPage:
   a. deleteExistingCartItems(), and
   b. verify_DeleteCartItems().

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

PraveenNamburi committed:
Developed a new test in catalog module - Verify_CheckOutDetails_Confirm.

LalithaBanda committed:
commiting unsaved changes

LalithaBanda committed:
Merged

LalithaBanda committed:
Commiting unsaved files

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/test/resources/catalog/cartCheckoutConfirmRequest/Verify_ConfirmRequestPage_AddComments.xml

LalithaBanda committed:
Developed new Test "Verify_ConfirmRequestPage_GrandTotal"

PraveenNamburi committed:
Added a new method in CartInformationPage - UpdateQuantityAndSaveCart().

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

LalithaBanda committed:
Moved Test Files to new Package "SearchFunctionalty"

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/SearchItems/LocalItemsTab.java

PraveenNamburi committed:
Developed the following tests in catalog module:
1. Verify_BackButton_CartCheckConfirmRequest,and
2. Verify_ConfirmRequest_AddMoreItems.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

LalithaBanda committed:
committing unsaved changes

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/main/java/com/xeeva/catalog/SearchItems/LocalItemsTab.java

LalithaBanda committed:
committing unsaved changes

LalithaBanda committed:
Developed new Tests "Verify_ConfirmRequest_AttachExternalFile",
Verify_ConfirmRequest_AttachInternalFile

PraveenNamburi committed:
1. Developed a new test in catalog module -
VerReq_CartCheckOutWithSameOrDifferentCC.
2. Added a new method in 'ConformRequestOPage' - clickbtnConfirm().
3. Added a new methods in CostCenterPage - 3.1
selectCCValueAtHeaderLevel(), and
3.2 verify_CostCenterUpdatedAtHeaderLevel().

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/ConfirmRequestPage.java

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

PraveenNamburi committed:
1.Created a new package for the latest test scripts and XML files -
'mainAppFlow'.

2. Developed a new test - VerReq_AddSmartFormItem_PerformCheckOut().

jessicamarshall commented on src/main/java/com/xeeva/catalog/CartInformationPage.java:
```diff
@@ -1,277 +1,318 @@
+	 * @author: Praveen Namburi, @version: Created 12-09-2016
+	 */
+	public void closeCartInfoPage(){
+		driver.setPageTimeout(2);
```
Please remove this page timeout

jessicamarshall commented on src/main/java/com/xeeva/catalog/CartInformationPage.java:
```diff
@@ -1,277 +1,318 @@
+			   }
+		       verify_UpdatedQuantity();
+			}
+			driver.setPageTimeout(3);
```
Please remove this page timeout

jessicamarshall commented on src/main/java/com/xeeva/catalog/CartInformationPage.java:
```diff
@@ -1,277 +1,318 @@
+			   }
+			   verify_DeleteCartItems();
+			 }
+		   driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
Unless you have changed the setElementTimeout from the default timeout time (Constants.ELEMENT_TIMEOUT), there is no need to change it back to the default.  That is what it is already.  Please remove this line.

Commit:
added javadocs

Commit:
Merge branch 'stage' of https://github.com/Orasi/Xeeva into stage

jessicamarshall commented on src/main/java/com/xeeva/catalog/CheckoutDetailPage.java:
```diff
@@ -1,66 +1,127 @@
+	 */
+	public void emailConfirmation(){
+		if(lnkEmailConfirmation.isDisplayed())
+			TestReporter.assertTrue(true, ""+lnkEmailConfirmation.getText().trim());
```
I think this needs updating.  So you are saying that if the email confirmation link is displayed then an assertion statement passes.  What if the email confirmation is not displayed - do you want the test to fail?  Additionally, are you also verifying that the email confirmation message is correct?  If so, then you'll need to compare the text against what you expect the message to say.  couple of options:
        TestReporter.assertTrue(lnkEmailConfirmation.isDisplayed(), "Verify that the email confirmation link displayed");
        TestReporter.assertEquals(lnkEmailConfirmation.getText().trim(),
                "The message you expect to display", "Verify the email confirmation message is correct");

jessicamarshall commented on src/main/java/com/xeeva/catalog/CheckoutDetailPage.java:
```diff
@@ -1,66 +1,127 @@
+		pageLoaded();
+		if(lblCartNumber.isDisplayed())
+			TestReporter.assertTrue(true, "Shopping Cart Number" + "  :  " + lblCartNumber.getText());
+	}
```
I think this needs updating.  Are you asserting that the cart number displays? Or are you asserting that the cart number is correct?  Or both.  Couple of options:
TestReporter.assertTrue(lblCartNumber.isDisplayed(), "Verify shopping cart number displayed");
TestReporter.assertEquals(lblCartNumber.getText(), "Shopping cart number you expect to display", 
                "Verify the shopping cart number is correct");

jessicamarshall commented on src/main/java/com/xeeva/catalog/ConfirmRequestPage.java:
```diff
@@ -1,98 +1,286 @@
+		pl.isDomComplete(driver);
+		btnBack.syncEnabled(30);
+		driver.executeJavaScript("arguments[0].click();", btnBack);
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
```
Unless you have changed the setElementTimeout from the default timeout time (Constants.ELEMENT_TIMEOUT), there is no need to change it back to the default. That is what it is already. Please remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/ConfirmRequestPage.java:
```diff
@@ -1,98 +1,286 @@
+			btnConfirm.syncVisible(5, false);
+			driver.executeJavaScript("arguments[0].click();", btnConfirm);
+			driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
+		}
```
Unless you have changed the setElementTimeout from the default timeout time (Constants.ELEMENT_TIMEOUT), there is no need to change it back to the default. That is what it is already. Please remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/ConfirmRequestPage.java:
```diff
@@ -1,98 +1,286 @@
+		driver.executeJavaScript("arguments[0].click();", Element);
+		btnBrowse.click();
+		try {Thread.sleep(3000);}catch (InterruptedException e){ e.printStackTrace();}
+
```
We have a sleeper class that does the Thread.sleep for you so you don't have to include the try/catch.  Please use instead.  

Sleeper.sleep(3000);

jessicamarshall commented on src/main/java/com/xeeva/catalog/ConfirmRequestPage.java:
```diff
@@ -1,98 +1,286 @@
+			robot.keyRelease(KeyEvent.VK_CONTROL);
+			robot.keyPress(KeyEvent.VK_ENTER);
+			robot.keyRelease(KeyEvent.VK_ENTER);
+
```
We will need to have a conversation about using the Robot class and a better strategy for this.  While this works locally on your machine - will not work when we execute these as a suite of test on the Jenkins machine - as the key presses will be on the jenkins machine and not done on the remote machines.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+	 */
+	public void AddInternalComment(String InternalComment){
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), then please remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+	public void change_CC(String changeType,String CC){
+		// after cart check out, application taking time to load cost center page 
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+		int loopCount=0;
+		for(WebElement nextMonth : nextMonthArrows){
+			nextMonth.click();
+			driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
So this line only sets the element timeout to the default (which it is already at).  If you need to change it from the default you will do something like this which will set it to 1 second -  'driver.setElementTimeout(1)' do the next lines of code, and then when finished, set it back to the default timeout - driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+			String randomID =input.getAttribute("name").replaceAll("\\D+", "");
+			String classValue = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[10]/a[2]/i")).getAttribute("class");
+			if(classValue.contains("cursor-pointer")){
+				TestReporter.assertTrue(true, "Edit Item for non Price Agreement Item is Enabled !!");
```
Is the purpose of this method to verify that the edit links are enabled?  If they are not enabled, do you need the test to fail?  If so it needs to be done like so:
TestReporter.assertTrue(classValue.contains("cursor-pointer"), "Verify the edit item for non price agreement is enabled");

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+		String randomID =unitPriceEdits1.get(1).getAttribute("id").replaceAll("\\D+", "");
+		String classValue = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[10]/a[2]/i")).getAttribute("class");
+		if(classValue.contains("opacity")){
+			TestReporter.assertTrue(true, "Edit Item for  Price Agreement Item is Disabled !!");
```
Same thing here:
TestReporter.assertTrue(classValue.contains("opacity"), "Verify the edit item for price agreement is disabled");

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+	 */
+	public void selectShippingAddress(){
+		pl.isDomComplete(driver);
+		lstShippingAdd.isEnabled();
```
If you are not doing anything with the return value of isEnabled(), then we need to remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+	}
+	
+	public String getCostCenterPageTitle(){
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,532 +1,583 @@
+		// Waiting for CC updated message to be visible. 
+		verify_CostCenterUpdatedAtHeaderLevel();
+		pageLoaded();
+		driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,221 +1,221 @@
+		if(lstSelectUOM.isDisplayed()){
+			lstSelectUOM.select(strUOMValue);
+			btnAddToCart.click();
+			driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,221 +1,221 @@
+	 */
+	public void addPriceAgreementItemToCart_And_Verify(){
+		btnAddToCart.jsClick();
+		driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,221 +1,221 @@
+
+	public void  modify_ItemDetails(String UP,String Qty,String UOM){
+		txtunitPrice.clear();
+		txtunitPrice.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), then please remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,221 +1,221 @@
+		modify_ItemDetails(UnitPrice,Quantity,strUOMValue);
+		btnAddToCart.isDisplayed();
+		btnAddToCart.click();
+		driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,221 +1,221 @@
+
+	public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMValue){
+		modify_ItemDetails(UnitPrice,Quantity,strUOMValue);
+		btnAddToCart.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), then please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ProductComparisonTab.java:
```diff
@@ -1,84 +1,84 @@
+		pageLoaded();
+		select_UOM(UOM);
+		driver.executeJavaScript("arguments[0].click();", lstAddToCart.get(0));
+		driver.setPageTimeout(3);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/ProductComparisonTab.java:
```diff
@@ -1,84 +1,84 @@
+		select_UOM(UOM);
+		driver.executeJavaScript("arguments[0].click();", lstAddToCart.get(0));
+		driver.setPageTimeout(3);
+		lblCartItemAddedMessage.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), then please remove this line.

jessicamarshall commented on src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java:
```diff
@@ -1,255 +1,255 @@
+		pageLoaded();
+		lnkItemNumber.syncVisible(10, false);
+		lnkItemNumber.click();
+		driver.setPageTimeout(3);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+		ReqTab.click();*/
+		ReqTab.syncVisible(30, false);
+		driver.executeJavaScript("arguments[0].click();", ReqTab);
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+		catalogSearch.sendKeys(searchItem);
+		searchButton.syncVisible(20,false);
+		searchButton.click();
+		driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
```
Please remove this line.  You are changing the timeout to 30 seconds after the button has already been clicked.  It does not help with waiting or page transitioning after you click a button.

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+				break;
+			}
+		}else if(getLocalItemsCount.contains("0")){
+			TestReporter.log(" 'No Records Found !!' in Local Items tab. "
```
Does this need to be a test failure instead of just reporting a log statement?

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+	 * @return getItemNumber
+	 */
+	public String  getItemNumberFromCatalog(){
+		driver.setPageTimeout(2);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+	 * @param cart, @param RFQ, @param OrderDesc, @param location
+	 */
+	public void enterSearchCriteriaAndVerifyRejectedOrders(String location){
+		driver.manage().timeouts().implicitlyWait(7, TimeUnit.SECONDS);
```
Remember - element timeouts only affect when using the commands driver.findBy and driver.findBys, and in turn the isDisplayed() and isEnabled() methods.  So if you are not using these methods, then there is no need to change the elmeent timeout.  
If you do need to change the element timeout, please use the orasi driver method: 
driver.setElementTimeout(7);
and then when you're done, to set back to the default:
driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+		lstLocation.select(location);
+		btnSearch.syncVisible(5);
+		driver.executeJavaScript("arguments[0].click();", btnSearch);
+		driver.manage().timeouts().implicitlyWait(6, TimeUnit.SECONDS);
```
Please remove line - see above comment.

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+				if(getRFQStatus.trim().contains("Cancelled")) break;  
+			}
+		}
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+					(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/tr["+row+"]/td[3]/span")).getText();
+			TestReporter.logStep("REQNumber is : "+getRFQStatus);
+			if(!getRFQStatus.contains("-")){
+				driver.setPageTimeout(2);
```
I think you need to do the element timeout here....pageTimeout only affects when using driver.get or driver.nagivate.  So you'll want to do 
driver.setElementTimeout(2);
and then when you're done, to set back to the default:
driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -1,763 +1,764 @@
+	 * @author  Praveen Varma @date 28/9/16
+	 */
+	public void getQtyNumber(){
+		String quantity = txtQuantityREQ.getAttribute("value");
```
Why are we just getting the quantity and not returning it?  There is no point in just logging to the reporter.

jessicamarshall commented on src/main/java/com/xeeva/catalog/SearchItems/GlobalItemsTab.java:
```diff
@@ -1,136 +1,138 @@
+	public void click_AddToCartButton(){
+		AddToCart.syncVisible(10, false);
+		AddToCart.click();
+		driver.setPageTimeout(3);
```
Please remove this line

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,317 +1,339 @@
+
+	public boolean isLogoutDisplayed() {
+		try {
+			Thread.sleep(3000);
```
Instead of doing a Thread.sleep surrounded by try/catch, let's use the Sleeper class instead.  So would be Sleeper.sleep(3000)

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,317 +1,339 @@
+
+	public void clickCartItemsLink() {
+		driver.executeJavaScript("arguments[0].click();", lnkCartItem);
+		driver.setPageTimeout(4);
```
Remove this line please

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,317 +1,339 @@
+		if (btnSaveCart.syncVisible(1, false)){
+			btnSaveCart.click();
+		} else {
+			TestReporter.logStep("Cart Empty to Save");
```
If the save button is not able to be clicked - do we need to fail instead of just reporting  log statement?

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,317 +1,339 @@
+				TestReporter.logStep( "Item Number Fronm Cart Rows : "+links.get(0).getText());
+				if(links.get(0).getText().equalsIgnoreCase(itemNumber)){
+					// clicking on Cart Item Edit
+					driver.setPageTimeout(3);
```
Remove this line please

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,317 +1,339 @@
+	// Method to Get Cart Items Count 
+	public int getCartItemsCount(){
+		pl.isDomComplete(driver);
+		driver.setPageTimeout(5);
```
Please remove this line

PraveenNamburi committed:
Added the files to index.

LalithaBanda committed:
Committing Unsaved Changes

LalithaBanda committed:
Developed new Test "VerReq_AddSmartFormItem_PerformCheckOut"

LalithaBanda committed:
Developed new page for approvals process

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/test/java/mainAppFlow/VerReq_AddSmartFormItem_PerformCheckOut.java

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

PraveenNamburi committed:
Created a new test data sheet for the test case -
VerifyBuyer_submitsRFQtoSuppliers.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

jessicamarshall commented on src/main/java/com/xeeva/approvals/ApprovalsPage.java:
```diff
@@ -0,0 +1,159 @@
+	public void click_ApprovalsTab(){
+		approvalsTab.syncVisible(30, false);
+		driver.executeJavaScript("arguments[0].click();", approvalsTab);
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
```
Please remove

Commit:
Added in method for attaching files without using system dialog

Commit:
quick commit

Commit:
quick fix after pull

PraveenNamburi committed:
Added a new workbook for the latest test case: TC_06 -
VerifyBuyer_submitsRFQtoSuppliers.

LalithaBanda committed:
Added new Methods

LalithaBanda committed:
Updated test "VerReq_addSmartFormItem_PerformCheckOut" by reading RFQ
value

LalithaBanda committed:
Developed new test "Verify_Vuyer_PreApprovalProcess"

LalithaBanda committed:
Developed new Test "VerifyApproverPerformAction"

LalithaBanda committed:
Added new Methods

LalithaBanda committed:
Commits after pull

LalithaBanda committed:
Resolved Conflicts

LalithaBanda committed:
commit unsaved changes

LalithaBanda committed:
Few fixes as per Review Comments

PraveenNamburi committed:
Added files to index.

LalithaBanda committed:
Removed Driver Quit, Updated BuyerProcess datasheet

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/approvals/ApprovalsPage.java
	src/main/java/com/xeeva/catalog/CheckoutDetailPage.java
	src/main/java/com/xeeva/catalog/ConfirmRequestPage.java
	src/main/java/com/xeeva/catalog/CostCenterPage.java
	src/main/java/com/xeeva/catalog/ItemDetailsPage.java
	src/main/java/com/xeeva/catalog/ProductComparisonTab.java
	src/main/java/com/xeeva/catalog/SearchItems/GlobalItemsTab.java
	src/main/java/com/xeeva/navigation/MainNav.java
	src/main/java/com/xeeva/quote/QuotePage.java
	src/test/java/catalog/cartCheckoutConfirmRequest/Verify_ConfirmRequest_AttachExternalFile.java
	src/test/java/catalog/cartCheckoutConfirmRequest/Verify_ConfirmRequest_AttachInternalFile.java
	src/test/java/mainAppFlow/VerifyApproverPerformAction.java
	src/test/resources/datasheets/VerifyBuyer_submitsRFQtoSuppliers.xlsx

PraveenNamburi committed:
Resolved the conflicts and updated the quotepage as it is.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/test/java/mainAppFlow/VerifyApproverPerformAction.java

PraveenNamburi committed:
Developed a new test - VerifyBuyer_submitsRFQtoSuppliers.

Commit:
added class diagrams

PraveenNamburi committed:
1. Updated the page classes as per the review comments provided by
jessica. 
2. Removed all the steps -
driver.manage().timeOuts().implicitlyWait(10,TimeUnit.SECONDS);

PraveenNamburi committed:
Updated the test data.

LalithaBanda committed:
Updated sheet

LalithaBanda committed:
added new methods

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	doc/ElementInternal.ucls

LalithaBanda committed:
commented page timeout

LalithaBanda committed:
committing unsaved files

LalithaBanda committed:
updated data provider input

LalithaBanda committed:
Merged

LalithaBanda commented:
Updated code as per review comments, 
Developed fallowing Testcases,
- VerReq_AddSmartFormItem_PerformCheckOut
- Verify_Buyer_PreApprovalProcess
- VerifyApproverAbleToPerformAction
- VerifyBuyer_submitsRFQtoSuppliers

LalithaBanda committed:
removed unused lines of code

PraveenNamburi committed:
Adding files to index.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

LalithaBanda committed:
Merge branch 'stage' of https://github.com/Orasi/Xeeva into lalitha

# Conflicts:
#	src/main/java/com/xeeva/quote/QuotePage.java
#	src/test/java/mainAppFlow/VerifyApproverAbleToPerformAction.java

PraveenNamburi committed:
Merge branch 'stage' of https://github.com/Orasi/Xeeva.git into stage

Conflicts:
	src/main/java/com/xeeva/catalog/CartInformationPage.java
	src/main/java/com/xeeva/catalog/ProductComparisonTab.java
	src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java
	src/main/java/com/xeeva/catalog/RequisitioningPage.java
	src/main/java/com/xeeva/catalog/SearchItems/BPOItemsTab.java
	src/main/java/com/xeeva/catalog/SearchItems/LocalItemsTab.java
	src/main/java/com/xeeva/login/LoginPage.java
	src/test/java/catalog/CancelRequisition_RecentOrdersList.java
	src/test/java/catalog/CreateSmartFormRequest.java
	src/test/java/catalog/VerifyCatalogSearch.java
	src/test/java/catalog/cartCheckoutProcess/ChangeQuantity_ExistingLineLevelRequisition.java
	src/test/java/catalog/cartCheckoutProcess/Verify_CopiedItem_LineLevel.java
	src/test/java/catalog/cartCheckoutProcess/Verify_ItemDelete.java

PraveenNamburi committed:
Added files to index.

LalithaBanda committed:
Updated sync timeouts

LalithaBanda committed:
Updated test steps

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/approvals/ApprovalsPage.java
	src/main/java/com/xeeva/catalog/RequisitioningPage.java
	src/main/java/com/xeeva/login/LoginPage.java
	src/main/java/com/xeeva/quote/QuotePage.java
	src/test/java/mainAppFlow/VerifyApproverAbleToPerformAction.java
	src/test/java/mainAppFlow/Verify_Buyer_PreApprovalProcess.java

LalithaBanda committed:
removed unused lines of code

PraveenNamburi committed:
Added files to index.

LalithaBanda committed:
added new method, updated Test steps , Updated Excel Data

LalithaBanda committed:
updated Test Steps

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/approvals/ApprovalsPage.java
	src/main/java/com/xeeva/login/LoginPage.java
	src/main/java/com/xeeva/quote/QuotePage.java
	src/test/java/mainAppFlow/VerifyApproverAbleToPerformAction.java
	src/test/java/mainAppFlow/VerifyBuyer_submitsRFQtoSuppliers.java

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/approvals/ApprovalsPage.java
	src/main/java/com/xeeva/quote/QuotePage.java
	src/test/java/mainAppFlow/VerifyApproverAbleToPerformAction.java
	src/test/java/mainAppFlow/VerifyBuyer_submitsRFQtoSuppliers.java

PraveenNamburi committed:
Added files to index

PraveenNamburi committed:
Updated the method - verify_SupplierIsAdded() in QuotePage.

PraveenNamburi committed:
Updated the method - verifySupplierIsAdded().

PraveenNamburi committed:
Removed the duplicated elements in quotepage.

LalithaBanda committed:
Added new Methods

LalithaBanda committed:
Updated Test Summary

LalithaBanda committed:
Added new test datatable,added environment,user credentials properties

LalithaBanda committed:
Developed new Page - Supplier
Developed new Test - Verify_RFQResponceBySupplier

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/login/LoginPage.java
	src/main/java/com/xeeva/quote/QuotePage.java
	src/test/java/mainAppFlow/Verify_Buyer_PreApprovalProcess.java

LalithaBanda committed:
commiting insaved changes

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/main/java/com/xeeva/approvals/ApprovalsPage.java
#	src/main/java/com/xeeva/catalog/RequisitioningPage.java
#	src/main/java/com/xeeva/login/LoginPage.java
#	src/main/java/com/xeeva/quote/QuotePage.java
#	src/test/java/mainAppFlow/VerifyApproverAbleToPerformAction.java
#	src/test/java/mainAppFlow/Verify_Buyer_PreApprovalProcess.java

LalithaBanda committed:
Commit for unsaved changes
